indexer/feature_visibility.cpp

```diff
@@ -196,7 +197,6 @@ namespace
     }
 
     static uint32_t const arrTypes[] = {
-      cl.GetTypeByPath({"wheelchair"}),
       cl.GetTypeByPath({"cuisine"}),
```

Comment:
why `fee` and `cuisine` are not moved also?

Reply:
These types are non-drawable only:
- _fee_ is not a searchable category
- _cuisine_ works with a separate code (not like categories)

Reply:
would be great to make `fee` searchable one day!